auto_round/export/export_to_autoround/export.py

```diff
@@ -89,15 +90,18 @@ def dynamic_QuantLienar_for_packing(backend, bits, group_size):
         return QuantLinear
     ##export all use trition, inference use exllamav2
     elif "autoround" in backend or "auto-round" in backend or "auto_round" in backend:
-        from auto_round_extension.cuda.qliner_triton import QuantLinear
-        return QuantLinear
+        if "qbits" in backend:
+            return qlinear_qbits.QuantLinear
+        else:
+            from auto_round_extension.cuda.qliner_triton import QuantLinear
+            return QuantLinear
 
     else:
         assert False, f"only support gptq and autoround backend"
 
 
 @register_format("auto_round")
-def save_quantized_as_autoround(output_dir, inplace=True, backend="autoround:exllamav2", **kwargs):
+def save_quantized_as_autoround(output_dir, inplace=True, backend="autoround:qbits", **kwargs):
```

Comment:
could qbits supports loading exllmav2 format model, I prefer to use exllamav2 as the default as it's commonly used in the community

Reply:
fix in [9287524](https://github.com/intel/auto-round/pull/145/commits/92875249b5a0dfb8602be5ebf2ee07fbced8dce2)